Makefile

```diff
@@ -51,6 +51,7 @@ firsttime_fsp:
 
 debiansysprepare:
 	sudo apt-get install device-tree-compiler pkg-config libssl-dev llvm-dev libclang-dev clang qemu-system-x86
+	sudo apt-get install libssl-dev
 	# --default-toolchain is purely an optimization to avoid downloading stable Rust first.
```

Comment:
Ah, I just saw that this is already installed in the line above, so this line can be removed as it's redundant.